src/app/components/CtaLink/index.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { withKnobs } from '@storybook/addon-knobs';
+import { withServicesKnob } from '../../legacy/psammead/psammead-storybook-helpers/src';
+import ThemeProvider from '../ThemeProvider';
+
+import CtaLink from '.';
+
+interface Props {
+  href: string;
+  children?: string;
+}
+
+const Component = ({ href, children }: Props) => (
+  <ThemeProvider service="news">
+    <CtaLink href={href}>{children}</CtaLink>
+  </ThemeProvider>
+);
+
+export default {
+  title: 'Components/CtaLink',
```

Comment:
Should we call it by its full name? (Also makes me think, should we also name the component CallToActionLink)?
```suggestion
  title: 'Components/Call To Action Link',
```

Reply:
👍 Done!